.cbmc-batch/jobs/aws_string_bytes/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags:        "--bounds-check;--pointer-check;--div-by-zero-check;--signed-overflow-check;--unsigned-overflow-check;--pointer-overflow-check;--undefined-shift-check;--float-overflow-check;--nan-check;--unwindset;memcmp.0:17,strlen.0:17,memcpy_impl.0:17,memset_impl.0:17"
```

Comment:
Unnecessary space at the beginning, same for all `cbmc-batch.yaml`

Reply:
See above